docs/resources/azure_monitor_activity_log_alert.md.erb

```diff
@@ -0,0 +1,83 @@
+---
+title: About the azure_monitor_activity_log_alert Resource
+platform: azure
+---
+
+# azure\_monitor\_activity\_log\_alert
+
+Use the `azure_monitor_activity_log_alert` InSpec audit resource to test properties
+of an Azure Monitor Activity Log Alert.
+
+<br />
+
+## Syntax
+
+An `azure_monitor_activity_log_alert` resource block identifies a Activity Log Alert by
+name and resource group.
+
+    describe azure_monitor_activity_log_alert(resource_group: 'example', name: 'alert-name') do
+      ...
+    end
+<br />
+
+## Examples
+
+### Test an example resource has an Activity Log Alert
+
+    describe azure_monitor_activity_log_alert(resource_group: 'example', name: 'alert-name') do
+      it { should exist }
+    end
+
+### Test an example resource has an Activity Log Alert with the correct opertion
+
+    describe azure_monitor_activity_log_alert(resource_group: 'example', name: 'alert-name') do
+      its('operations') { should include 'Microsoft.Authorization/policyAssignments/write' }
+    end
+
+<br />
+
+## Parameters
+
+* `name`
+* `resource_group`
+
+## Parameter Examples
+
+The resource group as well as the Activty Log Alert name.
+
+    describe azure_monitor_activity_log_alert(resource_group: 'example', name: 'alert-name') do
+      its('operations') { should include 'Microsoft.Authorization/policyAssignments/write' }
+    end
+
+## Properties
+
+* `operations`
+
+### operations
+
+The operation Administrative category and action of the Activity Log Alert
+
+    its('operations') { should include 'Microsoft.Authorization/policyAssignments/write' }
+
+## Matchers
+
+This InSpec audit resource has the following special matchers. For a full list
+of available matchers, please visit our [Universal Matchers page](https://www.inspec.io/docs/reference/matchers/).
+
+### exists
+
+The control will pass if the resource returns a result. Use `should_not` if you expect zero matches.
+
+    # example should always exist
+    describe azure_monitor_activity_log_alert(resource_group: 'example', name: 'alert-name') do
+      it { should exist }
+    end
+
+    # this activity log should not exist
```

Comment:
These comments look like nonsense because they refer to the same name ('alert-name'). Perhaps "If we expect 'alert-name' to always exist" and "If we expect 'alert-name' to never exist" instead?